tools/cli/flags.go

```diff
@@ -362,30 +366,28 @@ func getFlagsForStart() []cli.Flag {
 			Usage: "Optional input for the workflow from JSON file. If there are multiple JSON, concatenate them and separate by space or newline. " +
 				"Input from file will be overwrite by input from command line",
 		},
-		cli.StringFlag{
+		cli.StringSliceFlag{
 			Name:  FlagMemoKey,
-			Usage: "Optional key of memo. If there are multiple keys, concatenate them and separate by space",
+			Usage: fmt.Sprintf("Optional key of memo. If there are multiple keys, provide multiple %s flags", FlagMemoKey),
 		},
-		cli.StringFlag{
+		cli.StringSliceFlag{
 			Name: FlagMemo,
-			Usage: "Optional info that can be showed when list workflow, in JSON format. If there are multiple JSON, concatenate them and separate by space. " +
-				"The order must be same as memo_key",
+			Usage: fmt.Sprintf("Optional info that can be showed when list workflow. If there are multiple values, provide multiple %s flags. "+
+				"The order must be same as %s", FlagMemo, FlagMemoKey),
 		},
```

Comment:
I think it is worth to show an example of how to use with 2 keys and 2 values.

Reply:
Added example to PR description. I don't want to overload help screen as multiply flags with the same name will be a common pattern in new `tctl` and supposed to be well known.